### DISCUSSION TITLE
"url" parameter is valid but upstream response is invalid

### DISCUSSION THREAD

**Discussion by evestWebsite** [vercel/next.js#56004] [Help] [answered]
### Summary

Hi 
i use google cloud run to deploy my app , 
i deploy many versions on it all works fine , 
but suddenly my app is not showing images that i use in Image component 
and i get error 500 code. , and the respone is 
"url" parameter is valid but upstream response is invalid

**Comment by kutsan** [comment]
What's your Next.js version? I'm also having the same error.

**Reply by evestWebsite** [reply, marked as answer]
the issue was on verison 13.5 , so i downgrade to 13.4 and all is good now , try to downgrade next version to 13.4

**Comment by ShahriarKh** [comment]
see this issue: https://github.com/vercel/next.js/issues/53715

**Comment by stasdre** [comment]
Same problem in NextJS 14.0.0

**Reply by abouthugo** [reply]
yup this is still an issue for me in production environment
